docs/proposal-007-instance-selector.md

```diff
@@ -0,0 +1,364 @@
+# EC2 Instance Selector Support for Nodegroups
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+- [Proposal](#proposal)
+  - [Dry Run](#dry-run)
```

Comment:
should these be in two separate proposal files? It feels a bit hard to distinguish them atm

Reply:
They could be, but I'll have to reword some of the bits because right now it's written within the context of instance selector. I was planning to rename the title and filename to include dry-run, so I'll do that.